source/api/annex/notes/editors.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "IIIF Editorial Process"
+layout: spec
+tags: [annex, presentation-api, image-api]
+cssversion: 2
+---
+
+## Status of this Document
+{:.no_toc}
+
+This document is not subject to [IIIF's versioning semantics][iiif-semver]. Changes will be tracked within the document and its [internal change log][change-log]
+
+**Authors:**
+
+  * Michael Appleby, _Yale University_
+  * Tom Crane, _Digirati_
+  * Robert Sanderson, _Stanford University_
+  * Jon Stroop, _Princeton University_
+  * Simeon Warner, _Cornell University_
+  {: .names}
+
+{% include copyright.md %}
+
+----
+
+## Contents
+{:.no_toc}
+* Table of Discontent (will be replaced by macro)
+{:toc}
+
+## 1. Overview
+
+The IIIF editors are responsible for facilitating discussions across the IIIF community, digesting those discussions into use cases, and reflecting those use cases into new and revised specifications for the community to implement. This process is intended to be as transparent as possible. This document exists to further that intention by defining the IIIF editorial process and the responsibilities and expectations of the individual editors.
```

Comment:
> , ensuring those discussions are captured as use cases, 
as we don't want to be responsible for writing up all of the use cases, just making sure they're captured.